platform/marvell-armhf/sonic-platform-et6448m/entropy.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python3.5
```

Comment:
are we using python3.5 here? I do not remember we are using python3.5 on master image.

Reply:
we will get back to on this regarding python version.